<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testqtopencvClass</class>
 <widget class="QWidget" name="testqtopencvClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>testqtopencv</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>宽:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditWidth">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>640</string>
        </property>
        <property name="maxLength">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>高:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditHeight">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>480</string>
        </property>
        <property name="maxLength">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelFPS">
        <property name="text">
         <string>FPS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxGPU">
        <property name="text">
         <string>GPU</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonLoad">
        <property name="text">
         <string>加载图片</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCamera">
        <property name="text">
         <string>打开相机</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonVideo">
        <property name="text">
         <string>打开视频</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRefresh">
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDelete">
        <property name="text">
         <string>删除控件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAddOrigin">
        <property name="text">
         <string>原始图1</string>
        </property>
        <property name="type" stdset="0">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAddCascade">
        <property name="text">
         <string>检测2</string>
        </property>
        <property name="type" stdset="0">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAddBilateral">
        <property name="text">
         <string>双边滤波3</string>
        </property>
        <property name="type" stdset="0">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAddGaussian">
        <property name="text">
         <string>高斯4</string>
        </property>
        <property name="type" stdset="0">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAddWhite">
        <property name="text">
         <string>美白5</string>
        </property>
        <property name="type" stdset="0">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAddSharp">
        <property name="text">
         <string>锐化6</string>
        </property>
        <property name="type" stdset="0">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetContainer" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="testqtopencv.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonAddOrigin</sender>
   <signal>clicked()</signal>
   <receiver>testqtopencvClass</receiver>
   <slot>onAddWidget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAddCascade</sender>
   <signal>clicked()</signal>
   <receiver>testqtopencvClass</receiver>
   <slot>onAddWidget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAddBilateral</sender>
   <signal>clicked()</signal>
   <receiver>testqtopencvClass</receiver>
   <slot>onAddWidget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAddGaussian</sender>
   <signal>clicked()</signal>
   <receiver>testqtopencvClass</receiver>
   <slot>onAddWidget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRefresh</sender>
   <signal>clicked()</signal>
   <receiver>testqtopencvClass</receiver>
   <slot>onParamChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDelete</sender>
   <signal>clicked()</signal>
   <receiver>testqtopencvClass</receiver>
   <slot>onDeleteWidget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAddWhite</sender>
   <signal>clicked()</signal>
   <receiver>testqtopencvClass</receiver>
   <slot>onAddWidget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>714</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCamera</sender>
   <signal>clicked()</signal>
   <receiver>testqtopencvClass</receiver>
   <slot>onOpenCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAddSharp</sender>
   <signal>clicked()</signal>
   <receiver>testqtopencvClass</receiver>
   <slot>onAddWidget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>876</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonVideo</sender>
   <signal>clicked()</signal>
   <receiver>testqtopencvClass</receiver>
   <slot>onOpenVideo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAddWidget()</slot>
  <slot>onParamChanged()</slot>
  <slot>onDeleteWidget()</slot>
  <slot>onOpenCamera()</slot>
  <slot>onOpenVideo()</slot>
 </slots>
</ui>
